### PR TITLE
updating vision components environment

### PR DESCRIPTION
**Pull request by rjaincc**
updating vision components environment

### CODE REVIEW

**Review by novaturient95 — DISMISSED** [review]

File: assets/training/vision/components/image_classification/spec.yaml [hunk 1]
```diff
@@ -188,7 +188,7 @@ outputs:
 
 code: ../src
 
-environment: azureml:AzureML-AutoML-DNN-Vision-GPU:115
+environment: azureml://registries/azureml/environments/AzureML-AutoML-DNN-Vision-GPU/versions/117
```
Comment (PhaniShekhar) [inline assets/training/vision/components/image_classification/spec.yaml:191]: Why do we need to point to registry environment ? 
Wouldn't the CE from workspace env be picked up with the previous approach ?
If not, how did the runtime components already published by Gaurav/Daniel/Shubham work ?

**Review by PhaniShekhar — CHANGES_REQUESTED** [review]

File: assets/training/vision/components/image_classification/spec.yaml [hunk 2]
```diff
@@ -188,7 +188,7 @@ outputs:
 
 code: ../src
 
-environment: azureml:AzureML-AutoML-DNN-Vision-GPU:115
```
Comment (PhaniShekhar) [inline assets/training/vision/components/image_classification/spec.yaml:191]: Version 118 is being rolled out now, that has 1.52.0.post1. Could you please use that instead

**Review by vizhur — CHANGES_REQUESTED** [review]
these environments are NOT in the registry. Please consider to migrate them

File: assets/training/vision/components/image_classification/spec.yaml (hunk 1 above)
Comment (rjaincc) [reply assets/training/vision/components/image_classification/spec.yaml:191]: There are some recent changes in publishing pipelines, which is why these changes. 
Earlier, it used to work.

File: assets/training/vision/components/image_classification/spec.yaml (hunk 2 above)
Comment (rjaincc) [reply assets/training/vision/components/image_classification/spec.yaml:191]: Done

**Comment by rjaincc** [comment]
Closing, as changed have went in from another PR.

**Pull request closed without merging**